AMW_rest/src/main/java/ch/mobi/itc/mobiliar/rest/resources/ResourceTemplatesRest.java

```diff
@@ -105,7 +149,16 @@ public Response deleteResourceTemplate(@PathParam("resourceGroupName") String re
         return Response.status(Response.Status.OK).build();
     }
 
+    @DELETE
+    @Path("/{id : \\d+}")
+    @ApiOperation(value = "Remove a template")
+    public Response deleteTemplate(@PathParam("id") Integer id) throws TemplateNotDeletableException {
```

Comment:
Create a mapper for a more meaningful error message for TemplateNotDeletableException and add it to RESTApplication.